proposals/numpy.md

```diff
@@ -0,0 +1,162 @@
+## Introduction
+
+This project implements a subset of Numpy with focus on the most popular scalar and matrix operations. Two alternative solutions will be explored:
+
+- Run commands in WASM via [Pyodide](https://github.com/iodide-project/pyodide), which interfaces compiled Numpy packages in WASM.
```

Comment:
I'm also curious if we could install Pyodide separately, rather than checking in the whole thing (it's a big dependency!). Is it possible to just use the numpy part of Pyodide without importing all of it?

Reply:
I didn't find an existing approach to install the Numpy part of Pyodide separately. Will look around further. Thanks for all these suggestions professor!